### DISCUSSION TITLE
Windows - X to Tray

### DISCUSSION THREAD

**Discussion by Gamedirection** [rustdesk/rustdesk#8350] [General]
I would like to suggest an improvement in the user experience regarding RustDesk running in the background. It would be more convenient to have these apps located near the system tray instead of constantly open in the taskbar. For example, AltSnap allows for the option to hide the app from the tray, providing a cleaner desktop. I prefer to have certain apps running in the background without them being a distraction while I am using the computer. However, I often need to remote in from Google Remote Desktop to access these apps because I routinely close RustDesk to clear out my taskbar. If this functionality is already available, please let me know how to enable this. Thank you for your attention to this matter.

**Comment by rustdesk** [comment]
close as duplicate many times
